backend/src/backend/models.py

```diff
@@ -232,6 +232,9 @@ class Meta(BaseMeta):
 
     book: Book = ormar.ForeignKey(Book, related_name="metadata")
 
+    def __repr__(self):
+        return f"BookMetadata(book={repr(self.book)}"
```

Comment:
What information does this give? The name of the metadata is the most important.